server/server/src/test/kotlin/org/jetbrains/bsp/bazel/server/sync/BspProjectMapperTest.kt

```diff
@@ -0,0 +1,149 @@
+package org.jetbrains.bsp.bazel.server.sync
+
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
+import ch.epfl.scala.bsp4j.DependencyModule
+import ch.epfl.scala.bsp4j.DependencyModulesItem
+import ch.epfl.scala.bsp4j.DependencyModulesParams
+import ch.epfl.scala.bsp4j.DependencyModulesResult
+import ch.epfl.scala.bsp4j.MavenDependencyModule
+import ch.epfl.scala.bsp4j.MavenDependencyModuleArtifact
+import io.kotest.matchers.shouldBe
+import org.jetbrains.bsp.bazel.bazelrunner.utils.BazelRelease
+import org.jetbrains.bsp.bazel.server.model.Label
+import org.jetbrains.bsp.bazel.server.model.Library
+import org.jetbrains.bsp.bazel.server.model.Module
+import org.jetbrains.bsp.bazel.server.model.Project
+import org.jetbrains.bsp.bazel.server.model.SourceSet
+import org.junit.jupiter.api.Test
+import java.net.URI
+import java.nio.file.Paths
+
+class BspProjectMapperTest {
+  private val cacheLocation =
+    "file:///home/user/.cache/bazel/_bazel_user/ae7b7b315151086e31e3b97f9ddba009/execroot/monorepo/bazel-out/k8-fastbuild-ST-4a519fd6d3e4"
+
+  @Test
```

Comment:
I think you can add@Timeout annotation here to make the expectation more verbose and better documentation, [link for reference](https://junit.org/junit5/docs/5.5.1/api/org/junit/jupiter/api/Timeout.html)

Reply:
Added `@Timeout` annotation. However it doesn't seem to actually timeout at 30s under `bazel test`.